src/core/reopt_inputs.jl

```diff
@@ -215,18 +239,26 @@ end
 Create data arrays associated with techs necessary to build the JuMP model.
 """
 function setup_tech_inputs(s::AbstractScenario)
+    #TODO: new boiler, steam turbine, ghp
+    #TODO: create om_cost_per_kwh in here as well as om_cost_per_kw?
 
     techs = Techs(s)
 
     boiler_efficiency = Dict{String, Float64}()
-    production_factor = DenseAxisArray{Float64}(undef, techs.all, 1:length(s.electric_load.loads_kw))
 
     # REoptInputs indexed on techs:
     max_sizes = Dict(t => 0.0 for t in techs.all)
     min_sizes = Dict(t => 0.0 for t in techs.all)
     existing_sizes = Dict(t => 0.0 for t in techs.all)
     cap_cost_slope = Dict{String, Any}()
     om_cost_per_kw = Dict(t => 0.0 for t in techs.all)
+    production_factor = DenseAxisArray{Float64}(undef, techs.all, 1:length(s.electric_load.loads_kw))
+    tech_renewable_energy_pct = DenseAxisArray([1.0 for _ in 1:length(techs.all)], techs.all)#Dict(t => 1.0 for t in techs.all)
```

Comment:
@hdunham were you thinking of changing all these DenseAxisArrays to Dicts (as shown in the comments)? I think that is a good idea since I think that DenseAxisArrays are unnecessary and add complexity

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/28/commits/e0d97c036284e70ae76b909c43daf48696b9a895